src/styles/main.scss

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
+  @import "variables";
```

Comment:
All imports should be made at the beginning of the file and they should not be placed in specific tags.